backend/alerts/integrations/webhook/webhook.go

```diff
@@ -0,0 +1,141 @@
+package webhook
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/hashicorp/go-retryablehttp"
+	"github.com/highlight-run/highlight/backend/alerts/integrations"
+	"github.com/highlight-run/highlight/backend/model"
+	e "github.com/pkg/errors"
+	"net/http"
+)
+
+type ErrorAlertWebhook struct {
+	Event string
+	*integrations.ErrorAlertPayload
+}
+
+func sendWebhookData(destination string, body []byte) error {
+	resp, err := retryablehttp.Post(destination, "application/json", body)
```

Comment:
I'm trying to find it in their [docs](https://github.com/hashicorp/go-retryablehttp) but it's not really standing out -- what is the retry logic this uses? i.e. how many times does it retry and what is the duration between each retries?

Reply:
here's the spec for their default client - thought it was pretty reasonable.

<img width="562" alt="Screenshot 2023-03-24 at 12 38 43 PM" src="https://user-images.githubusercontent.com/1351531/227623460-4b39b932-54c8-4302-bdfb-95793674cb1b.png">

Reply:
One more thing --- have we considered any authentication?

Reply:
i feel like that would be more of a V2 for #4550 - allowing customizing the webhook payload and sending a request with authentication

Reply:
as discussed offline, i'll make the model for webhooks an object so that we can store an auth token and other payload configuration in the future.